ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -75,6 +75,82 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 2 * * *
```

Comment:
Any specific reason to have a different cron expression in respect to the default job (every 4 hours)?

Reply:
We could run it more often but I was thinking just once per day was enough. I'm thinking we might have a few variations of baremetal deployments that we want to test daily/weekly just to make sure it still works (IPv4, external DHCP, virtual media, etc).

Reply:
Sure ok, sort of less frequent configuration